_data/changelogs/template-documentation.yml

```
@@ -0,0 +1,16 @@
+title: Documentation page
+type: template
+items:
+  - date: NNNN-NN-NN
+    summary: Documentation page structure follows logical order.
+    summaryAdditional: Sidenav in template now follows logical HTML order on mobile. Previously, the sidenav was shifted after the content, which did not follow the HTML order.
```

Added a few suggestions to follow standard structure and be a bit more plain language. 
```suggestion
    summary: Updated the visual order of the documentation page to match the HTML order. 
    summaryAdditional: Now, the side navigation is visually displayed before the main content at narrow screen widths. 
```

---

Updated in afb926f.

Thanks for the suggestions. I wanted to keep the summary short, but modified it for clarity based on your recommendation. Added your recommendation for `summaryAdditional`.